packages/app-plugin-security-cognito/src/identityProvider/index.tsx

```diff
@@ -51,7 +51,11 @@ export default (): PluginCollection => [
                 <Cell span={12}>
                     <Bind
                         name="password"
-                        validators={data.createdOn ? undefined : validation.create("required")}
+                        validators={
+                            data.createdOn
+                                ? validation.create("minLength:8")
+                                : validation.create("required,minLength:8")
```

Comment:
For this one, I'd like @Pavel910 to chime in.

I believe it's not smart to hardcode this here because this rule can change in the Cognito settings. Am I right?

Maybe we could just add a parameter here (`apps/admin/code/src/plugins/security.ts:27`):

![image](https://user-images.githubusercontent.com/5121148/113395039-8d031180-9399-11eb-97d8-cb2df58bfd03.png)

Of course, if not specified, then use the `minLength:8` or whatever default Cognito setting is.

Reply:
This is what password policy can look like:
![image](https://user-images.githubusercontent.com/3920893/113411915-4c19f580-93b7-11eb-807d-13a2dbbf5db1.png)

So we should support:
```
{
                minimumLength: 8,
                requireLowercase: false,
                requireNumbers: false,
                requireSymbols: false,
                requireUppercase: false,
 }
```
and generate validators based on that.

Reply:
@Pavel910 I've updated the PR with the suggested upgrade. Please feel free to review.
Thanks :+1: